src/MacroExpander.js

```diff
@@ -200,6 +184,46 @@ class MacroExpander {
                 }
             }
         }
+
+        // Flow unable to figure out that this pathway is impossible.
+        // https://github.com/facebook/flow/issues/4808
+        throw new Error(); // eslint-disable-line no-unreachable
+    }
+
+    /**
+     * Returns the expanded macro as a reversed array of tokens and a macro
+     * argument count.
+     * Caches macro expansions for those that were defined simple TeX strings.
+     */
+    _getExpansion(name: string): ParsedExpansion {
+        const lookup = this.macros[name];
```

Comment:
Maybe `definition` would be a more apt variable name.  Then maybe `MacroExpansion` could be renamed to `MacroDefinition`.  I think this will better help separate the concepts of expanding macro definitions from the definitions themselves.  Also, renaming `ParsedExpansion` should probably be renamed to `MacroExpansion` as the parser hasn't done anything with the tokens yet.

TL;DR:
```
lookup -> definition
MacroExpansion -> MacroDefinition
ParsedExpansion -> MacroExpansion
```

Reply:
Done.